package.json

```diff
@@ -139,13 +139,14 @@
     "versioned": "npm run prepare-test && time ./bin/run-versioned-tests.sh"
   },
   "bin": {
-    "newrelic-naming-rules": "./bin/test-naming-rules.js"
+    "newrelic-naming-rules": "bin/test-naming-rules.js"
   },
   "dependencies": {
-    "@grpc/grpc-js": "1.1.8",
+    "@grpc/grpc-js": "^1.2.0",
```

Comment:
Are you intentionally not pinning the version now? We havent' really discussed if we are ready to do that yet or not.

Reply:
Good catch.  I'll pin that.